### PR TITLE
add modal|add login page|add routing

### PR DESCRIPTION
**Pull request by anastasiialukash**

### CODE REVIEW

File: server/server.js [hunk 1]
```diff
@@ -16,6 +17,7 @@ app.get('/', function (req, res) {
 
 app.use('/directories', directoriesRouter)
 app.use('/notices', noticesRouter)
```
Comment (unnamed92) [inline server/server.js:19]: needs formatting

File: src/App.js [hunk 2]
```diff
@@ -1,11 +1,23 @@
...
-import {SidebarPanel} from "./components/SidebarPanel/SidebarPanel";
+import { BrowserRouter, Route, Routes } from 'react-router-dom';
+import {Login} from "./components/Login/Login";
+import {useState} from "react";
 
 function App() {
-  return (
-      <ManagerContainer />
-  );
+    const [token, setToken] = useState();
```
Comment (unnamed92) [inline src/App.js:8]: it could be placed in separate hook useToken

File: src/components/CreateNoteButton/CreateNoteButton.jsx [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+            <Fab sx={{display: 'right'}} color="primary" aria-label="add" onClick={openModal}>
+                <AddIcon/>
+            </Fab>
+            <Modal
+                isOpen={modalIsOpen}
+                onRequestClose={closeModal}
+                style={customStyles}
+                contentLabel="Example Modal"
+            >
+                <div style={{
```
Comment (unnamed92) [inline src/components/CreateNoteButton/CreateNoteButton.jsx:66]: non conditional inline styling

File: src/components/CreateNoteButton/CreateNoteButton.jsx [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+import axios from 'axios';
+import Fab from '@mui/material/Fab';
+import AddIcon from '@mui/icons-material/Add';
+import Modal from 'react-modal';
+import Button from '@mui/material/Button';
+import Stack from '@mui/material/Stack';
+
+export const CreateNoteButton = () => {
+    const [description, setDescription] = useState(null);
+    const [modalIsOpen, setIsOpen] = React.useState(false);
```
Comment (unnamed92) [inline src/components/CreateNoteButton/CreateNoteButton.jsx]: use useState or React.useState

File: src/components/NoteDirectory/NoteDirectory.jsx [hunk 5]
```diff
@@ -1,56 +1,64 @@
...
+import axios from "axios";
+
+const noteIcon = <svg xmlns="http://www.w3.org/2000/svg" height="24" viewBox="0 0 24 24" width="24"><path d="M0 0h24v24H0V0z" fill="none"/><path d="M22 10l-6-6H4c-1.1 0-2 .9-2 2v12.01c0 1.1.9 1.99 2 1.99l16-.01c1.1 0 2-.89 2-1.99v-8zm-7-4.5l5.5 5.5H15V5.5z"/></svg>;
+const dirClosed = <svg xmlns="http://www.w3.org/2000/svg" height="24" viewBox="0 0 24 24" width="24"><path d="M0 0h24v24H0z" fill="none"/><path d="M10 4H4c-1.1 0-1.99.9-1.99 2L2 18c0 1.1.9 2 2 2h16c1.1 0 2-.9 2-2V8c0-1.1-.9-2-2-2h-8l-2-2z"/></svg>;
 
 export const NoteDirectory = () => {
-    const data = [
-        {id: "1", name: "Unread"},
-        {id: "2", name: "Threads"},
+    //todo: map data from the server to dirs format
```
Comment (unnamed92) [inline src/components/NoteDirectory/NoteDirectory.jsx:10]: delete unused code

File: src/index.css [hunk 6]
```diff
@@ -13,4 +13,10 @@ body {
 code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
   monospace;
+}
+
+.login-wrapper {
```
Comment (unnamed92) [inline src/index.css]: index.css for reseting styles, not for styling components

File: src/components/NoteDirectory/NoteDirectory.jsx (hunk 5 above)
Comment (anastasiialukash) [reply src/components/NoteDirectory/NoteDirectory.jsx:10]: this is todo, will change this part later

File: src/App.js (hunk 2 above)
Comment (anastasiialukash) [reply src/App.js:8]: created a task for it, will do later

**Pull request merged**